python/README.md

```diff
@@ -1,19 +1,13 @@
 ## Running the Example
 
 - [Python 3.6 or above is required](https://www.python.org/downloads/)
-- [Setting up virtual environments is recommended](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/)
-
 
 ```bash
-python3 -m venv momento-examples-env
-
-source momento-examples-env/bin/activate
 
-pip install -r requirements.txt --extra-index-url https://momento.jfrog.io/artifactory/api/pypi/pypi-public/simple
+python3 -m pip install -r requirements.txt --extra-index-url https://momento.jfrog.io/artifactory/api/pypi/pypi-public/simple
 
 MOMENTO_AUTH_TOKEN=<YOUR_TOKEN> python example.py
```

Comment:
NIT: when I tried this on my laptop I needed `python3 example.py` otherwise it ran my default python 2 and didn't find the momento module
```suggestion
MOMENTO_AUTH_TOKEN=<YOUR_TOKEN> python3 example.py
```

Reply:
On that note - this doesn't work with Python 2? I thought both Python 2 and 3 are in stable support? Again, very n00b here

Reply:
Python 2.7 is now sunset https://www.python.org/doc/sunset-python-2/ . Some of our internal dependencies only support python3. Hence, I decided to call this out. We can revisit feasibility for deprecated version, based on customer needs.

Reply:
> NIT: when I tried this on my laptop I needed `python3 example.py` otherwise it ran my default python 2 and didn't find the momento module

Erm. yes I missed this. Thank you for spotting this.